packages/web-react/src/components/UNSTABLE_Header/useUnstableHeaderStyleProps.ts

```diff
@@ -10,14 +11,21 @@ export interface HeaderStyles<T> {
 }
 
 export const useUnstableHeaderStyleProps = (props: SpiritHeaderProps): HeaderStyles<SpiritHeaderProps> => {
+  const { hasBottomDivider = false, ...restProps } = props;
+
   const headerClass = useClassNamePrefix('UNSTABLE_Header');
   const headerLogoClass = useClassNamePrefix('UNSTABLE_HeaderLogo');
+  const headerHasBottomDividerClass = useClassNamePrefix('UNSTABLE_Header--hasBottomDivider');
```

Comment:
I am not sure if we do it like this or like this
```suggestion
  const headerHasBottomDividerClass = `${headerClass}--bottomDivider`;
  ```